tests/Aspire.Playground.Tests/Infrastructure/DistributedApplicationExtensions.cs

```diff
@@ -89,6 +89,7 @@ public static Task WaitForResource(this DistributedApplication app, string resou
         return app.ResourceNotifications.WaitForResourceAsync(resourceName, targetState, cancellationToken);
     }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
nit: can you place these just around `KnownResourceStates.Hidden` in the method instead of the entire method